service/src/test/java/uk/gov/hmcts/reform/fpl/service/DateFormatterServiceTest.java

```diff
@@ -42,6 +60,24 @@ void shouldFormatLocalDateTimeInExpectedFormat() {
         assertThat(formattedDate).isEqualTo("12:00pm, 1 January 2019");
     }
 
+    @ParameterizedTest
+    @MethodSource(value = "dayOfMonthSuffixSource")
+    void shouldReturnExpectedSuffixWhenGivenAValidDay(int[] days, String expected) {
+        for (int day : days) {
+            String suffix = dateFormatterService.getDayOfMonthSuffix(day);
+            assertThat(suffix).isEqualTo(expected);
+        }
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, 32})
+    void shouldErrorWhenGivenAValueSmallerThan1AndGreaterThan31(int day) {
```

Comment:
shouldThrow

Reply:
SmallerThan1OrGreated, maybe just when day of month is invalid

Reply:
Yeah that sounds better, names are hard